recipes/pdoc/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "pdoc" %}
+{% set version = "11.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/mitmproxy/pdoc/archive/refs/tags/v{{ version }}.tar.gz
+  # pdoc does not currently upload a distrobution (tar.gz) file to PyPi, so I'm
```

Comment:
This is not a big deal. In fact, sometimes it is desirable b/c most of the time the GH tarball is more "complete."

Reply:
the pdoc maintainer just added tarballs to pypi, so I still might switch. Main reason I prefer pypi is because of the autotick bot. But do you know if the autotick bot works for github releases too?

Reply:
Yep. Both the bot and the recipe generator, grayskull, works with GH tarballs.